### PR TITLE
[hypodermic] added version 2.5.3, updated baseline

### PR DESCRIPTION
**Pull request by helynranta**
**Describe the pull request**

- #### What does your PR fix?  
This PR adds new version 2.5.3, which contains bug fix for nested containers

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
I have tested new version with windows and macOS. 
I updated baseline.

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
I believe so. I only added new version.

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
Yes

### CODE REVIEW

File: ports/hypodermic/portfile.cmake [hunk 1]
```diff
@@ -1,8 +1,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO ybainier/Hypodermic
-    REF 3e86a5a1fd5e8279d6ca461f9f398fa3f3c2eddc # v2.5.2
-    SHA512 1af2a94037aa5bf8c65aceb4a2e941f7f6d016422d345f86280085115e9bb871387370158b1a83891be8efdadd4eea0a1f8905225ebee64c000ec9023a9f212e
+    REF 0e0d85d70aa2f2391dfd84f8af4a3863d4fb1611 # v2.5.3
+    SHA512 6fc3f9eca034a4de3f7086bd51e9ba11ee31c8ec000a3e0bdfc06db1f3c12a89b66793adf5d219441e680541e26acaef72d21f9dd0acf3f5fee3aa12d3fb7b4d
     HEAD_REF master
     PATCHES
         "disable_hypodermic_tests.patch"
```
Comment (NancyLi1013) [inline ports/hypodermic/portfile.cmake:8]: Could you please handle license like this?
`file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)`

Please also add quotes for other paths. Please see  https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/cmake-guidelines.md.

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: ports/hypodermic/portfile.cmake (hunk 1 above)
Comment (lerppana) [reply ports/hypodermic/portfile.cmake:8]: is this better?

Comment (NancyLi1013) [reply ports/hypodermic/portfile.cmake:8]: Yes, it it good now.

**Review by NancyLi1013 — APPROVED** [review]

**Comment by BillyONeal** [comment]
I couldn't find documentation suggesting that hypodermic follows semver so I changed to "version".

**Review by BillyONeal — APPROVED** [review]
Will merge as soon as build comes back.

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**